src/constants.js

```diff
@@ -18,7 +18,13 @@ export const DIMENSION_KEYS = {
   total: "total",
 };
 
-const DIMENSIONS = {
+export const DATA_KEY_TRANSLATIONS = {
```

Comment:
i have a renaming suggestion for this: `DIMENSION_DATA_KEYS` ... I think it's good to be a little more specific about what kind of keys we are describing here